docs/csharp/tour-of-csharp/tutorials/arrays-and-collections.md

```diff
@@ -22,7 +22,7 @@ Create a directory named *list-tutorial*. Make that the current directory and ru
 Open *Program.cs* in your favorite editor, and replace the existing code with the following:
 
 ```csharp
-var names = new List<string> { "<name>", "Ana", "Felipe" };
+List<string> names = [ "<name>", "Ana", "Felipe" ];
```

Comment:
All the other updates formatted without the leading and ending space. I'm only suggesting this for consistency, but it looks like a lot of these would need to be updated to format like this: `[x, y, z]`.

Reply:
Will update. What's interesting is that the VS refactoring left the spaces in.